merfi/collector.py

```diff
@@ -1,15 +1,16 @@
 from __future__ import with_statement
 import os
 import re
+import merfi
 
 
 class FileCollector(list):
 
-    def __init__(self, config=None, _eager=True):
-        config = config or {}
+    def __init__(self, path=None, config=None, _eager=True):
```

Comment:
this makes it easier to use the FileCollector, passing the path directly is better for testing as well.